packages/@okta/courage-for-signin-widget/README.md

```diff
@@ -0,0 +1,8 @@
+generates a Courage version for signin-widget
+
+## Steps
+
+0. make sure VPN is connected
+1. upgrade courage version. e.g. `yarn upgrade @okta/courage@4.4.0-beta.xxxxxx`
+2. `yarn build`
```

Comment:
Do we want to mix `yarn` and `npm`? At the repository root, we use `npm` for all build/start instructions.

As a future task, lets update all Sign-In Widget docs to use `yarn` instead of `npm`.

Reply:
I guess I'm used to use `yarn` (especially my intention for this package is for internal).
I could change it to `npm` or if we're comfortable to use `yarn` at the repo root, I'll create a follow up task to consolidate all npm usage to yarn plus the package-lock.json to yarn.lock.

Reply:
@manueltanzi-okta has #285  already for this, lets get him to close it out :)